README.md

```diff
@@ -1,25 +1,26 @@
 # veryVerilog
 
-veryVerilog enables an easy way to program PICs from family **PIC16F180XY** and **PIC16F131XY** over a Web Browser without the need to install any software using a PIC as a USB HiD device. 
+veryVerilog enables an easy way to program PICs from family **PIC16F180xy** and **PIC16F131xy** over a Web Browser without the need to install any software using a PIC as a USB HiD device. 
 
 The project is based on 3 parts:
 - USB-HID PCB (hardware)
 - USB-HID Hex File (firmware)
 - Web Page to flash Hex files a PIC device attached to the hardware
 
-The PCB looks like this and is based on a [**MCP2221A**](https://www.microchip.com/en-us/product/mcp2221a)
+The PCB looks like this and is based on a [**PIC16F1455**](https://www.microchip.com/en-us/product/pic16f1455)
```

Comment:
Either you place a point and keep on going the phrase or you make a new line.

Reply:
Added dot and newline. Looks better. The dot is intentionally separated from the part# to minimize confusion.